crates/ev-cli/src/fs.rs

```diff
@@ -0,0 +1,256 @@
+use serde::{Deserialize, Serialize};
```

Comment:
lifted directly, updated error enum to use thiserror (not all of these functions are used yet but will be in other functions commands)